src/Sylius/Bundle/CoreBundle/spec/Checkout/CheckoutProcessScenarioSpec.php

```diff
@@ -51,7 +52,10 @@ function it_builds_checkout_process_with_proper_steps(ProcessBuilderInterface $b
         $builder->setDisplayRoute('sylius_checkout_display')->willReturn($builder)->shouldBeCalled();
         $builder->setForwardRoute('sylius_checkout_forward')->willReturn($builder)->shouldBeCalled();
 
-        $builder->setRedirect('sylius_homepage')->willReturn($builder)->shouldBeCalled();
+        $builder->setRedirect('sylius_account_order_show')->willReturn($builder)->shouldBeCalled();
+        $builder->setRedirectParams(array(
+            'number' => $order->getNumber()
+        ))->willReturn($builder)->shouldBeCalled();
```

Comment:
This is wrong, cause [mock (aka predictions)](http://www.phpspec.net/en/latest/manual/prophet-objects.html#mocks) should be called __only__ when you don't care about what is being returned by the method, if you care about it you should call [stubs (aka promises)](http://www.phpspec.net/en/latest/manual/prophet-objects.html#stubs).

As said calling both is wrong & mixing behavior of two different things.